### ISSUE TITLE
Feature Request: Abort task while running

### ISSUE THREAD

**Issue by shahar4499** [cuckoosandbox/cuckoo#1379]
It would be useful if I could get cuckoo to abort a single task in the middle of an analysis.
Now you have to either restart cuckoo or wait until the task is finished.
If I set a timeout for a task, cuckoo would wait until it's reached, even if I turn off the machine in the middle of it.

**Comment by jbremer** [comment]
Yes, thanks for the request. This, among several other operations to affect the throughput of analyses etc, is on our todo list somewhere, so please stay tuned :)
